src/vs/workbench/contrib/comments/browser/commentsViewActions.ts

```diff
@@ -17,17 +17,23 @@ import { COMMENTS_VIEW_ID } from 'vs/workbench/contrib/comments/browser/comments
 import { FocusedViewContext } from 'vs/workbench/common/contextkeys';
 import { viewFilterSubmenu } from 'vs/workbench/browser/parts/views/viewFilter';
 
+export type CommentsSortOrder = 'resource' | 'updatedAt';
```

Comment:
I would prefer an enum with string values here so that when you reference `'updatedAt'`, in the context key expression (currently on line 25) you can use the enum to ensure there are no typos in the string.

Reply:
Won't string literal unions achieve the same thing? With the existing code any invalid string value would be considered a TS error.

I'd be happy to change this though, just curious if there's something I might have missed.